packages/snjs/lib/utils.spec.ts

```diff
@@ -0,0 +1,13 @@
+import createDOMPurify from 'dompurify';
+import { JSDOM } from 'jsdom';
+
+const window = new JSDOM('').window;
+const DOMPurify = createDOMPurify(window as never);
+
+describe.only('utils', () => {
+  it('sanitizeHtmlString', () => {
+    const dirty = '<svg><animate onbegin=alert(1) attributeName=x dur=1s>';
+    const cleaned = DOMPurify.sanitize(dirty);
```

Comment:
Shouldn't we test the exported function `sanitizeHtmlString` instead?

Reply:
Tests don't have a `window` object so require configuration with JSDom. Utils.ts doesn't require this however.